_includes/footer.html

```diff
@@ -2,15 +2,13 @@
 	<div class="zerogrid">
 		<div class="wrap-footer">
 			<div class="row">
-				<div class="col-1-2">
-					<div class="wrap-col" style="color:#595959">
-						<span>社團法人中華慈雲慈善利生推廣協會</span><br/>
-						<span>會址 : 11149台北市士林區平等里平菁街95巷47號之2</span><br/>
-						<span>連絡電話 : (02)-28620012 , (02)-28610043 傳真 : (02)-28621852</span><br/>
-						<span>Email : fayu991010@gmail.com</span><br/>
-						<span>辦事處 : 23857新北市樹林區八德街90號2樓(法雨講堂)</span><br/>
-						<span>聯絡電話 : (02)-86845058  傳真電話 : (02)-86845028</span><br/>
-					</div>
+				<div style="color:#595959; margin:30px;">
+					<span>社團法人中華慈雲慈善利生推廣協會</span><br/>
+					<span>會址 : 11149台北市士林區平等里平菁街95巷47號之2</span><br/>
+					<span>連絡電話 : (02)-28620012 , (02)-28610043 傳真 : (02)-28621852</span><br/>
+					<span>Email : fayu991010@gmail.com</span><br/>
+					<span>辦事處 : 23857新北市樹林區八德街90號2樓(法雨講堂)</span><br/>
+					<span>聯絡電話 : (02)-86845058  傳真電話 : (02)-86845028</span><br/>
```

Comment:
If you want to describe a list, use tag `<dl>` and `<dt>`, see http://www.w3schools.com/tags/tag_dl.asp.
`color:#595959` is not needed, looks like the default text color already set to `#595959`?